packages/shopify-codemod/transforms/remove-return-voids.js

```diff
@@ -0,0 +1,34 @@
+import {matchLast} from './utils';
+
+export default function removeReturnVoids({source}, {jscodeshift: j}, {printOptions = {}}) {
+
+  return j(source)
+    .find(j.Function, {
+      body: {
+        body: matchLast(j, {
+          type: 'IfStatement',
+          alternate: {
+            body: matchLast(j, {
+              type: 'ReturnStatement',
+            }),
+          },
+        }),
+      },
+    })
+    .filter(({node: {body: {body}}}) => {
+      const ifBlock = body[body.length - 1].alternate.body;
+      const returnStatement = ifBlock[ifBlock.length - 1];
+      let isUndefined = false;
+      if (
+      returnStatement.argument.operator === 'void' && returnStatement.argument.argument.raw === '0' ||
+      returnStatement.argument.name === 'undefined'
+      ) {
```

Comment:
I have a check for whether something is `undefined` or not here: https://github.com/Shopify/javascript/blob/master/packages/shopify-codemod/transforms/coffeescript-soak-to-condition.js#L279-L288, might be worth pulling that out into `utils.js` and importing it in both spots.